### DISCUSSION TITLE
Start fullscreen games on other/second screen or monitor? (Linux-Ubuntu)

### DISCUSSION THREAD

**Discussion by w-sky** [Heroic-Games-Launcher/HeroicGamesLauncher#2727] [Ideas]
Hello, I am using a notebook with an external monitor attached as two-screen setup and all games that start in fullscreen mode will open on the internal screen, however I would like to use the external monitor.

Same problem might occur with a desktop PC and two monitor setup: There seems to be no way to choose the default screen where to launch games on in Heroic Games Launcher.

**Comment by w-sky** [comment]
My workaround is to switch off the internal screen via system settings when playing a game, but that's rather tedious and like to use the internal screen (or second screen) sometimes for a browser or chat without closing the game.

**Reply by w-sky** [reply]
Sometimes it's also possible to switch the game from fullscreen to windowed, then move the window to the right screen, then switch back to fullscreen. However, this does not work always or it may be necessary each time I start the game. So still very tedious.

**Comment by srgzx** [comment]
Hi, have you been able to figure out any better workaround for this?  I'm on MacOS and I have the same problem.  There's no way to make the game go fullscreen on monitor 2.

**Reply by w-sky** [reply]
> Hi, have you been able to figure out any better workaround for this? I'm on MacOS and I have the same problem. There's no way to make the game go fullscreen on monitor 2.

Unfortunately I did not find a way to use the 2 screen setup properly, other than described before.

**Comment by umutyazgan** [comment]
I'm currently having the same issue on my 2 screen desktop setup. The game always opens in my vertical monitor and I can't move it to my main one.

OS: Arch Linux

DE: KDE Plasma

Heroic version: 2.16.1

**Comment by Aylawoo** [comment]
OS: EndeavourOS
DE: KDE Plassma 6
Heroic version: 2.18.1

I am begging, someone, anyone, please tell me how to get this working. Death Stranding for example opens on a monitor that is physically behind me. Disconnecting/reconnecting monitors every time I open a game is not an option. Windowed mode does not work, cannot drag the window. Keyboard shortcuts to move the window do not work. Setting the game to use the correct monitor DOES move it to the right monitor...but only in windowed mode, and then turning it back to fullscreen doesn't get rid of the Windows 95 title bar, and all mouse input is ~250 pixels offset down and to the right of where the cursor actually is.

How can anyone say with a straight face that gaming on linux works? This is a terrible experience. There has to be a solution, right?

**Reply by Aylawoo** [reply]
I have finally figured out a workaround for this. As far as I can tell, this is the best possible solution until someone figures out a clever way to actually force monitors:

Meta+W...you can drag fullscreen applications between screens in the overview on Plasma. Why this suggestion was not on any forum, git issue, reddit thread, or anywhere else I looked I have no idea. I found this by accident by keysmashing, and then had to figure out what the keyboard shortcut was by scrolling through all the bindings in the settings menu.

Meta+W.

It's that easy. 

Obviously this only works in Plasma, I know GNOME has a similar overview though (Whether it works or not), other desktops will probably be a tossup.
